common/client-core/src/client/base_client/storage/migration_helpers.rs

```diff
@@ -2,210 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 pub mod v1_1_33 {
-    use crate::client::base_client::{
```

Comment:
don't change old migration code. you have to create your own version and then add additional path there